tests/e2e/etcd.go

```diff
@@ -58,10 +58,6 @@ func NewEtcd(ctx *TestCtx, optMods ...EtcdOptModifier) (*Etcd, error) {
 	return etcd, nil
 }
 
-func (ec *Etcd) Stop(options ...interface{}) error {
-	return ec.ComponentRuntime.Stop(nil, options)
```

Comment:
Etcd container still need to be stopped and removed, right? So the ComponentRuntime.Stop is here still needed here.

Reply:
There is no context cleanup needed for the etcd container, so the ComponentRuntime.Stop function can be used directly via interface embedding.

P.S. The e2e tests were failing because I forgot to change the cleanup of the microservice component (this should be fixed in f69d594).